ABFIAE-41:

Smiilesx3 opened the pull request:
added ENUM LevelOfHungriness
added Testanimal method

changed feedAnimals/doAction method

Smiilesx3 committed:
ABFIAE-41:
added ENUM LevelOfHungriness
added Testanimal method

changed feedAnimals/doAction method

shinydesteny1512 commented on src/main/farmyard/Shed/ChickenShed.java:
```diff
@@ -17,7 +18,15 @@ public void addChicken(Chicken chicken) {
     public void chickenAction() {
         for (Chicken chicken : chickenList) {
-            chicken.doAction();
+            if (chicken.getLevelOfHungriness() == LevelOfHungriness.OVEREAT.getHungriness()) {
```
== zu ..getLevelOfHungriness().equelsIfnoreCase(...);

shinydesteny1512 commented on src/main/farmyard/Shed/ChickenShed.java:
```diff
@@ -30,14 +39,18 @@ public void actionSound() {
-                System.out.println(chicken.getName() + "was fed");
-            } else {
-                System.out.println(chicken.getName() + "isn't hungry");
+            if (chicken.getLevelOfHungriness() == LevelOfHungriness.HUNGRY.getHungriness()) {
```
== zu ..getLevelOfHungriness().equelsIfnoreCase(...);

shinydesteny1512 commented on src/main/farmyard/Shed/CowShed.java:
```diff
@@ -18,7 +18,15 @@ public void addCow(Cow cow) {
     public void cowAction() {
         for (Cow cow : cowList) {
-            cow.doAction();
+            if (cow.getLevelOfHungriness() == LevelOfHungriness.OVEREAT.getHungriness()) {
```
== zu ..getLevelOfHungriness().equelsIfnoreCase(...);

shinydesteny1512 commented on src/main/farmyard/Shed/CowShed.java:
```diff
@@ -32,16 +40,20 @@ public void actionSound() {
-            } else {
-                System.out.println(farmAnimal.getName() + "isn't hungry");
+        for (Cow cow : cowList) {
+            if (cow.getLevelOfHungriness() == LevelOfHungriness.HUNGRY.getHungriness()) {
```
== zu ..getLevelOfHungriness().equelsIfnoreCase(...);

shinydesteny1512 commented on src/main/farmyard/Shed/DogShed.java:
```diff
@@ -25,23 +26,35 @@ public void dogAction() {
     public void actionSound() {
         for (Dog dog : dogList) {
-            dog.makeSound();
+            if (dog.getLevelOfHungriness() == LevelOfHungriness.OVEREAT.getHungriness()) {
```
== zu ..getLevelOfHungriness().equelsIfnoreCase(...);

shinydesteny1512 commented on src/main/farmyard/Shed/DogShed.java:
```diff
@@ -25,23 +26,35 @@ public void dogAction() {
-            } else {
-                System.out.println(farmDog.getName() + "isn't hungry");
+        for (Dog dog : dogList) {
+            if (dog.getLevelOfHungriness() == LevelOfHungriness.HUNGRY.getHungriness()) {
```
== zu ..getLevelOfHungriness().equelsIfnoreCase(...);

shinydesteny1512 commented on src/main/farmyard/Shed/PigShed.java:
```diff
@@ -17,7 +18,15 @@ public void addPig(Pig pig) {
     public void pigAction() {
         for (Pig pig : pigList) {
-            pig.doAction();
+            if (pig.getLevelOfHungriness() == LevelOfHungriness.OVEREAT.getHungriness()) {
```
== zu ..getLevelOfHungriness().equelsIfnoreCase(...);

shinydesteny1512 commented on src/main/farmyard/Shed/PigShed.java:
```diff
@@ -32,15 +41,19 @@ public void actionSound() {
-                System.out.println(pig.getName() + "was fed");
-            } else {
-                System.out.println(pig.getName() + "isn't hungry");
+            if (pig.getLevelOfHungriness() == LevelOfHungriness.HUNGRY.getHungriness()) {
```
== zu ..getLevelOfHungriness().equelsIfnoreCase(...);

shinydesteny1512 commented on src/main/farmyard/Shed/DogShed.java:
```diff
@@ -25,23 +26,35 @@ public void dogAction() {
     // dog makes sound
     public void actionSound() {
         for (Dog dog : dogList) {
```
action sound und doAction ???????

Smiilesx3 committed:
ABFIAE-41:

changed feedAnimals "==" changed to equalsIgnoreCase
doAction method "==" changed to equalsIgnoreCase
adjusted dogAction() and actionSound()

cgi-nheinemann committed:
ABFIAE-32

Database connection created
and added some methods
